SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -207,6 +226,7 @@ private extension PersonController {
         backgroundFistImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
         setupTableView()
         setupDonationBottomView()
+        view.addSubview(viewBlur)
     }
```

Comment:
If you extract all the other logic into its own function, this can be placed at the top of the function and removed from here

Reply:
Leave this here because i want the view to be blurred immediately when the view is loaded. My function isn't called until   fetch details has been called